readthedocs/builds/version_slug.py

```diff
@@ -30,7 +30,7 @@
 #   +? -- allow multiple of those, but be not greedy about the matching
 #   (?: ... ) -- wrap everything so that the pattern cannot escape when used in
 #                regexes.
-VERSION_SLUG_REGEX = '(?:[a-z0-9][-._a-z0-9]+?)'
+VERSION_SLUG_REGEX = '(?:[a-z0-9][-._a-z0-9]*?)'
```

Comment:
My initial intend for adding the "non-greedy" matcher was that the regex does not leak into some other bits of a URL regex when it is embedded into one. But, yeah, I think it won't make any difference in real use. Shall I remove it?